docsets/Kubernetes/docset.json

```diff
@@ -1,10 +1,20 @@
 {
     "name": "Kubernetes",
-    "version": "1.4",
+    "version": "1.7.5",
     "archive": "Kubernetes.tgz",
     "author": {
         "name": "John Patterson",
         "link": "https://twitter.com/cantrobot"
     },
     "aliases": ["kubernetes", "kubectl"]
+    "specific_versions": [
+        {
+            "version": "1.7.5",
+            "archive": "versions/1.7.5/Kubernetes.tgz"
+        },
+        {
+            "version": "1.4",
+            "archive": "versions/1.4/Kubernetes.tgz.txt"
```

Comment:
This doesn't work. You need to add an actual `tgz` at this location.

Reply:
I can't find the original archive anymore.

Reply:
You can grab it from https://kapeli.com/feeds/zzz/user_contributed/build/Kubernetes/Kubernetes.tgz.

Reply:
cool. fixed and rebased

Reply:
The docset.json still needs to be updated to it points to the `tgz`, and not `tgz.txt`.

Reply:
Fixed :(